mlir/lib/Target/IRDLToCpp/Templates/DialectDecl.txt

```diff
@@ -0,0 +1,28 @@
+R"(
```

Comment:
Out of curiosity, would it be possible to reuse this file between ODS and this tool, or are there missing feature that would currently prevent that?